Demos/Shopping/ShoppingUITests/TestPages/FavoritesPage.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FavoritesPage.swift
+//  ShoppingUITests
+//
+//  Created by Albert Bori on 1/6/23.
+//
+
+import XCTest
+
+/// The test view for the favorites list which provides the ability to remove favorites
+struct FavoritesPage: PushedPage, TabbedPage {
+    let app: XCUIApplication
+    let previousView: AccountTabPage
+    
+    init(app: XCUIApplication, previousView: AccountTabPage, file: StaticString = #file, line: UInt = #line) {
+        self.app = app
+        self.previousView = previousView
+        XCTAssertTrue(app.collectionViews.element.waitForExistence(), "Can't find Favorites nav bar item", file: file, line: line)
```

Comment:
here you use XCTAssert and below you are using your own assert. Is there a reason for that?

Reply:
There was no particularly good reason to use one over the other here. I have made the calls more consistent in the latest push to this PR.